package.json

```diff
@@ -7,7 +7,7 @@
   "homepage": "https://nebenan.de/",
   "repository": "good-hood-gmbh/nebenan-form",
   "bugs": "https://github.com/good-hood-gmbh/nebenan-form/issues",
-  "version": "7.0.1",
+  "version": "7.0.2-beta.0",
```

Comment:
looks good, just publish as regular 7.0.2 changes are minor and beta not needed for testing